.travis.yml

```diff
@@ -0,0 +1,38 @@
+language: rust
+sudo: false
+
+matrix:
+  include:
+
+  # tests pass
+  - rust: stable
+    script:
+    - cargo test --locked
+    - rustup component add rustfmt-preview
+    - cargo fmt --all -- --check
+    env: RUST_BACKTRACE=1
+
+  # dist linux binary
+  - env: TARGET=x86_64-unknown-linux-musl DEPLOY=1
+    rust: stable
+    before_script:
+      - rustup target add $TARGET
+      - curl https://www.openssl.org/source/openssl-1.0.2l.tar.gz | tar xzf -
+      - (cd openssl-1.0.2l &&
+        CC=musl-gcc ./Configure --prefix=$HOME/openssl-musl no-dso no-ssl2 no-ssl3 linux-x86_64 -fPIC &&
+        make -j$(nproc) &&
+        make install)
```

Comment:
Is this truly necessary? Or is it actually cargo cult? ;) (In both cases it'd be great to add a comment so I don't as you again in 2 months)

Reply:
lol cargo cult :P will tear out in a sec